conversions/decimal_to_binary.py

```diff
@@ -1,25 +1,43 @@
-"""Convert a Decimal Number to a Binary Number."""
+"""Convert ANY Decimal Number to a Binary Number."""
 
+def dtbconverter(num):
+    """ Function inputs a float value and returns a list as output """
+    decimal_accuracy = 7
```

Comment:
Please consider changing this to __def decimal_to_binary(num: float, decimal_accuracy: int = 7) -> float:__  The current function name is more descriptive, the type hints will help callers and mypy and allowing callers to mess with __decimal_accuracy__ has more value than hardcoding it.

Also, please consider adding some __doctests__ with  ints, floats, zero, negative numbers, and for error checking, an empty list and a string.